src/main/java/uk/gov/hmcts/dts/fact/controllers/admin/list/AdminAreasOfLawController.java

```diff
@@ -0,0 +1,46 @@
+package uk.gov.hmcts.dts.fact.controllers.admin.list;
+
+import java.util.List;
+import static org.springframework.http.ResponseEntity.ok;
+import static uk.gov.hmcts.dts.fact.services.admin.AdminRole.FACT_ADMIN;
+import static uk.gov.hmcts.dts.fact.services.admin.AdminRole.FACT_SUPER_ADMIN;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.hmcts.dts.fact.config.security.Role;
+import uk.gov.hmcts.dts.fact.model.admin.AreaOfLaw;
+import uk.gov.hmcts.dts.fact.services.admin.list.AdminAreasOfLawService;
+
+
+@RestController
+@RequestMapping(
+    path = "/admin/courtAreasOfLaw",
```

Comment:
This is getting all areas of law and not specific to a single court. I wonder if it is better to rename the path to simply "/admin/areasOfLaw" ?